rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/SQSRestServerBuilder.scala

```diff
@@ -141,7 +142,8 @@ case class TheSQSRestServerBuilder(providedActorSystem: Option[ActorSystem],
         changeMessageVisibilityBatch(p) ~
         deleteQueue(p) ~
         getQueueAttributes(p) ~
-        setQueueAttributes(p)
+        setQueueAttributes(p) ~
+        addPermissions(p)
```

Comment:
That's it! Thank you very much! IntelliJ hides the file extensions in the project tree so i didn't see that there was none, now everything compiles just fine and i can finally start to work on the function. Thanks again.